decidim-participatory_processes/spec/system/admin/admin_import_participatory_process_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require "spec_helper"
+require "tempfile"
```

Comment:
Why do you need to require the `tempfile` and `json` dependencies here? I don't see them used anywhere.

I also tried running this spec with these two lines commented and it passed fine.